pages/task/read/views/index.jsx

```diff
@@ -70,23 +71,24 @@ const Task = ({ task }) => {
       <div className="govuk-grid-row">
         <div className="govuk-grid-column-one-third">
           <ol className="section-navigation">
-            <li className="active"><a href="#">Some</a></li>
-            <li><a href="#">Navigation</a></li>
-            <li><a href="#">Here</a></li>
+            <li className="active"><a href="#training">Training and exemptions</a></li>
+            <li><a href="#procedures">Procedure Categories</a></li>
+            <li><a href="#endorse">Endorse Application</a></li>
           </ol>
         </div>
 
         <div className="govuk-grid-column-two-thirds">
-          <Form formatters={formatters} />
+          <Pil />
+          <div id="endorse">
+            <Form formatters={formatters} />
+          </div>
```

Comment:
This looks a bit odd, that we have PIL specific stuff in the left column, and then a `<PIL />` component in the right column.

It would probably be better to have the `<PIL />` component deal with all the PIL things across two columns, and then a separate row/columns underneath for the common form part.

Reply:
I'm happy for this to be left until the next implementation though, and not resolved in this PR.

Reply:
We'll probably want a `SectionNavigation` component at some point that you can pass section props to (map of labels & section ids?) that can trigger a scroll to the section - it's gonna be needed for the NTS review page, and probably other bits as well.